Google.PowerShell/Logging/GcLogMetricCmdlets.cs

```diff
@@ -156,12 +122,17 @@ public class NewGcLogMetricCmdlet : GcLogEntryCmdletWithLogFilter
 
         /// <summary>
         /// <para type="description">
-        /// The description of the metric to be created.
-        /// </para> 
+        /// The description of the metric.
+        /// </para>
         /// </summary>
         [Parameter(Mandatory = false, Position = 1)]
         public string Description { get; set; }
 
+        /// <summary>
+        /// Given a log metric body, returns either a create or update request.
+        /// </summary>
+        protected abstract LoggingBaseServiceRequest<LogMetric> GetRequest(LogMetric logMetric);
```

Comment:
I think this comment is a little confusing.  This just returns a LoggingBaseServiceRequest right?  It's not two separate request types?